src/style.css

```diff
@@ -1 +1,91 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header,
+.nav_list {
+  display: flex;
+  align-items: center;
+}
+
+.header {
+  font-family: inherit;
+  font-weight: 500;
+  font-style: normal;
+  justify-content: space-between;
+  position: relative;
+  height: 60px;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05) ;
+}
+
+.header__link {
```

Comment:
Consider removing the unnecessary `display: flex;` since it's not needed when styling the header link.

Reply:
I made all the changes but when i remove flex from the moyo icon it goes slightly higher and two tests are failing . So i just kept it on display flex.